dGame/UserManager.cpp

```diff
@@ -323,7 +324,7 @@ void UserManager::CreateCharacter(const SystemAddress& sysAddr, Packet* packet)
 	}
 
 	//Now that the name is ok, we can get an objectID from Master:
-	ObjectIDManager::RequestPersistentID([=, this](uint32_t objectID) {
+	ObjectIDManager::RequestPersistentID([=, this](uint32_t objectID) mutable {
```

Comment:
Anyway to avoid the mutable specification here? I'm fine with it if there's no other option, but...

Reply:
no other option, need the objid for the name and that's only gotten inside the lambda